packages/embark/src/lib/modules/coverage/contractEnhanced.ts

```diff
@@ -32,13 +32,14 @@ export class ContractEnhanced {
 
   constructor(public filepath: string, public solcVersion: string) {
     this.id = nextId();
+    this.source = fs.readFileSync(filepath, "utf-8");
+    this.originalSource = this.source;
```

Comment:
Not sure how moving this out of the `try/catch` helps? What was happening before? Do we want it to be caught by a higher up `try/catch`?

Reply:
On the `catch` it was setting source and originalSource to blank for no reason. I'm probably going to burn in hell for this, but I don't think it will have problems reading contracts from disk without other things also being catastrophic. Really the only issue I foresee here is that the parser will fail.

Again, don't quote me on this, and I'll categorically deny I've ever said what I just said.

Reply:
Ah ok, So we can put the read back in the `try` then?

Reply:
I guess we could, yeah.